libbeat/scripts/Makefile

```diff
@@ -405,11 +409,15 @@ package: package-setup
 	fi
 
 	SNAPSHOT=${SNAPSHOT} BUILDID=${BUILDID} BEAT_DIR=${BEAT_DIR} BUILD_DIR=${BUILD_DIR} $(MAKE) -C ${ES_BEATS}/dev-tools/packer ${PACKAGES} ${BUILD_DIR}/upload/build_id.txt
-
+	$(MAKE) fix-permissions
 	echo "Finished packages for ${BEATNAME}"
 
 package-dashboards: package-setup
 	mkdir -p ${BUILD_DIR}
 	cp -r _meta/kibana ${BUILD_DIR}/dashboards
 	# build the dashboards package
 	BEATNAME=${BEATNAME} BUILD_DIR=${BUILD_DIR} SNAPSHOT=$(SNAPSHOT) $(MAKE) -C ${ES_BEATS}/dev-tools/packer package-dashboards ${shell pwd}/build/upload/build_id.txt
+
+fix-permissions:
+	# Change ownership of all files inside /build folder from root/root to current user/group
+	docker run -v ${BUILD_DIR}:/build -it alpine sh -c "chown -R $(shell id -u):$(shell id -g) /build"
```

Comment:
I would prefer if we use a "fixed" version of alpine here. We had it in the past that an update to a docker image broke the builds.